apps/web/src/components/dashboard/energy-consumption-card.tsx

```diff
@@ -66,7 +100,7 @@ export default async function EnergyConsumptionCard({ startDate, endDate, aggreg
                             <p className="text-muted-foreground">In diesem Zeitraum stehen keine Daten zur Verfügung</p>
                         </div>
                     ) : (
-                        <EnergyConsumptionCardChart data={aggregatedData} peaks={peaks} />
+                        <Chart />
                     )}
```

Comment:
Allgemein: Ich habe jetzt heute das Ticket "Weitere aggregierte Werte" in main gepusht. Da musst du mal schauen, ob es dadurch jetzt zu Konflikten gekommen ist

Reply:
Ne, gibt keine Probleme